sematic/plugins/metrics_storage/sql/models/metric_label.py

```diff
@@ -33,3 +33,5 @@ class MetricLabel(Base):
     metric_name: str = Column(types.String(), nullable=False)
     metric_labels: MetricsLabels = Column(JSONB(), nullable=False)
     metric_type: MetricType = Column(IntEnum(MetricType), nullable=False)
+
+    __table_args__ = (Index("metric_labels_name_labels_idx", "metric_name", "metric_labels", unique=True), )
```

Comment:
compound indexes with unique constraints must be added via `__table_args__` in declarative sqlalchemy models

Reply:
Follow-up task: Please add a check that validates that the code and the provided db are in sync.
This can be checked on server startup & migration startup, and warn in the logs if they mismatch, in order to inform users they have issues. The warning can point to a documentation entry on what to do to remedy the situation. Most likely to contact us.
This can be added as a CI test to validate we don't introduce discrepancies in the future.